### PR TITLE
Show banner if the user has claimed already

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

Show banner message if the user has no more available claims but has already claimed some claims
Screenshot from 2022-02-04 14-21-57

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @nenadV91 , great!
However, some cases: 
1. As a not connected user, I see that an account has available tokens to claim in the GC, but the account has already claimed. However, reported in #2382 , and might need a more complex solution to solve this...
image
image

2. I'd rewrite the message to make it simpler. Something like 'This account has already claimed vCOW tokens!' or so.

3. As for the colors, I think we have a lot of different ones on the Claim form for warning/info messages. So as a proposal, I'd use the same color for the banner as we do for the warning on the last claim page. @biocom , WDYT?
image
image
image

File: src/custom/pages/Claim/ClaimBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import { ClaimCommonTypes } from 'pages/Claim/types'
+
+type ClaimBannerProps = Pick<ClaimCommonTypes, 'hasClaims'> & {
+  isClaimed: boolean
+}
+
+export default function ClaimBanner({ hasClaims, isClaimed }: ClaimBannerProps) {
+  const { claimStatus, activeClaimAccount, isInvestFlowActive } = useClaimState()
+
+  const isEligible =
```
Comment (alfetopito) [inline src/custom/pages/Claim/ClaimBanner.tsx:16]: I think `isEligible` name no longer applies. Maybe `shouldShowBanner`?

**Review by alfetopito — COMMENTED** [review]
While it does communicate the message to the user in the banner, the rest of the page says something different:

Screen Shot 2022-02-04 at 08 13 59

What about hiding all that (`0 vCOW` and the text)?
Or maybe load the vCOW balance for this account and say something like:

```
This account has now:
[cow logo] XXX vCOW

Welcome to Cowmunity etc...
```

Also the button at the bottom has no relevant action and should just be removed

File: src/custom/pages/Claim/ClaimBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+  if (!isEligible) return null
+
+  return (
+    <ClaimBannerWrapper isClaimed={isClaimed}>
+      <SVG src={CheckCircle} description="eligible" />
+      <Trans>
+        {hasClaims
+          ? 'This account is eligible for vCOW token claims!'
+          : 'This account has claimed available vCOW token claims!'}
```
[the remark this answers is not in the document]
Comment (nenadV91) [reply src/custom/pages/Claim/ClaimBanner.tsx]: Updated this with Elena's suggestion as we said in the sync.

File: src/custom/pages/Claim/ClaimBanner.tsx (hunk 1 above)
Comment (nenadV91) [reply src/custom/pages/Claim/ClaimBanner.tsx]: Updated

**Comment by nenadV91** [comment]
@alfetopito we said yesterday on sync that we will just show the banner for this PR and then probably add your suggested changes as a following PR.

**Review by elena-zh — APPROVED** [review]
LGTM now!

**Review by alfetopito — APPROVED** [review]

**Review by alfetopito — APPROVED** [review]

**Pull request merged**